ipywidgets/widgets/tests/test_datetime_serializers.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Vidar Tonaas Fauske.
```

Comment:
Looks like a couple of these headers here and below could be updated to match the existing ones.